app/models/manageiq/providers/openstack/cloud_manager.rb

```diff
@@ -31,6 +31,8 @@ class ManageIQ::Providers::Openstack::CloudManager < EmsCloud
 
   include ManageIQ::Providers::Openstack::ManagerMixin
 
+  validates :authentications, :presence => {:message => 'You must enter credentials.'}
```

Comment:
Hrm, if we're going to do this, then it probably makes more sense to do this at either the [`cloud_manager`](https://github.com/ManageIQ/manageiq/blob/3447f98fed91bb8d015f6ffc9755c9bd7d27e47a/app/models/manageiq/providers/cloud_manager.rb) level or [`base_manager`](https://github.com/ManageIQ/manageiq/blob/3447f98fed91bb8d015f6ffc9755c9bd7d27e47a/app/models/manageiq/providers/base_manager.rb) level.

Otherwise, we're going to get different behavior across different providers for credentials requirements.

/cc @Fryguy @juliancheal @jrafanie 

Thoughts?